core-bundle/src/Resources/contao/themes/flexible/src/main.css

```diff
@@ -1012,6 +1012,9 @@ label.tl_change_selected {
 	height:16px;
 	vertical-align:-3px;
 }
+#tl_buttons + #tl_breadcrumb {
+	margin:18px 18px 0 18px;
```

Comment:
Why is this necessary? There should be CSS definitions for `#tl_breadcrumb` already.

Reply:
Because the breadcrumb has no margin. The margin inherits from the tl_formbody_edit, that is not present here.

without the css / with the css definition
<img width="943" alt="Screen Shot 2019-08-21 at 16 44 56" src="https://user-images.githubusercontent.com/1284725/63442301-25ea5080-c433-11e9-95d7-8088fafa06c0.png">
<img width="931" alt="Screen Shot 2019-08-21 at 16 45 11" src="https://user-images.githubusercontent.com/1284725/63442302-25ea5080-c433-11e9-9aef-ff036b9f7e0d.png">

Reply:
In Contao 4.8 the definition should be `.content > nav` 😅